fundingmanager.go

```diff
@@ -1047,6 +1047,7 @@ func (f *fundingManager) handleFundingOpen(fmsg *fundingOpenMsg) {
 		FundingFeePerKw: 0,
 		PushMSat:        msg.PushAmount,
 		Flags:           msg.ChannelFlags,
+		MinConfs:        1,
```

Comment:
In this case we don't _really_ need to set the value here as we're responding to a request.

Reply:
Yeah I figured I'd add it just to prevent it from being forgotten to be done once we support dual funding.